extensions/ql-vscode/src/helpers.ts

```diff
@@ -232,6 +232,15 @@ export function getOnDiskWorkspaceFolders() {
   return diskWorkspaceFolders;
 }
 
+/** Check if folder is already present in workspace */
+export function isFolderAlreadyInWorkspace(folderPath: string) {
```

Comment:
Are we guaranteed that `folderPath` as well as al the workspace folders are absolute and normalized?

Reply:
@shati-patel and I had a chat about this. We're not sure how to guarantee that we always know what the path contains so we decided it's simpler to go back to checking for the folder _name_ is present in the list of folders in the workspace instead of the path. 

I'll add a note to mention this decision on the ticket and ask James whether he has any opinions.